mobidam-sst-management-backend/src/main/java/de/muenchen/mobidam/repository/ZuordnungRepository.java

```diff
@@ -0,0 +1,11 @@
+package de.muenchen.mobidam.repository;
+
+import de.muenchen.mobidam.domain.Zuordnung;
+import java.util.UUID;
+import org.springframework.data.repository.CrudRepository;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public interface ZuordnungRepository extends CrudRepository<Zuordnung, UUID> {
```

Comment:
Ein Unittest mit einer testSave-Methode für das Repo wäre schön.

Reply:
Eigentlich kenne ich es als Good Practice die Funktionalitäten zu testen, die wir auch implementieren. Bei einem testSave Unittest würden wir Spring Boot testen und sonst nichts. Da sehe ich den Mehrwert nicht. Aber es ist auch nur meine Meinung 😄

Reply:
In dem Fall ist es auch mehr ein Integrationstest als ein Unittest. Und im Archetypen ist das ja für TheEntity auch drin. Ich gehe also davon aus, dass es CommonSense der Community ist, das zu machen.